src/main/java/snippets/Snippets.java

```diff
@@ -38,6 +38,26 @@
 
 public abstract class Snippets {
 
+
+    /** Input a line of numbers separated by space as integers
+     * and return ArrayList of Integers.
+     * eg. the String "1 2 3 4 5 6 7 8 9" is returned as an ArrayList of Integers.
+     * 
+     * @param numbers range of numbers separated by space as a string
+     * @return ArrayList of Integers
+     */
+    
+    public static ArrayList<Integer> inputInt(String numbers)
```

Comment:
1. Can we have a better method name? For example, stringToInts
2. Please return `int[]` instead of ArrayList
3. Move this method to bottom of the file

Reply:
Please update `README.md` appropriately as well. This method will go under String section.